modules/ROOT/pages/errors/gql-errors/42I05.adoc

```diff
@@ -0,0 +1,4 @@
+= 42I05
+
+== Status description
+error: syntax error or access rule violation - invalid FIELDTERMINATOR. The FIELDTERMINATOR specified for LOAD CSV can only be one character wide. The FIELDTERMINATOR specified for LOAD CSV can only be one character wide.
```

Comment:
The sentence 

The FIELDTERMINATOR specified for LOAD CSV can only be one character wide.

is duplicated by misstake. This was already wrong before the split, but I didn't notice it until now. If we decide to keep the status descriptions in the index file as well, it should be updated in both places.